src/plays/travel-advisor/styles/dropdown.css

```diff
@@ -0,0 +1,52 @@
+select {
```

Comment:
@yung-coder , do not directly change the style of the element, I would recommend using class.